src/main/resources/application.properties

```diff
@@ -10,3 +10,10 @@ spring.servlet.multipart.max-request-size=10MB
 file.upload-dir=${FILE_UPLOAD_DIRECTORY:./images}
 file.ddl-auto=create-drop
 configuration.mock-data=true
+
+
+#  =========================  #
+#  ======  JWT CONFIG  ======  #
+application.security.jwt.secret=f7a44a02a4932ce59a078f5a08cc790df0900c5bd9d6cc7218c78f8d3df6ddb7
```

Comment:
Mozna od razu dac `application.security.jwt.secret=${JWT_SECRET:f7a44a02a493...}`, zeby umozliwic potem dodawanie tego przez zmienna srodowiskowa. Ale to nie musi byc teraz